starter-code/app.js

```diff
@@ -22,6 +25,7 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', index);
+//app.use('/beach', beach);s
```

Comment:
Same that before. Furthermore, keep in mind that, if you put /beach, all the routes in beach.js are going to expect /beach/whatever-you-put-in-the-route.